itez/beneficiary/api/views.py

```diff
@@ -0,0 +1,61 @@
+from rest_framework.mixins import ListModelMixin, RetrieveModelMixin, UpdateModelMixin, CreateModelMixin
+from rest_framework.viewsets import GenericViewSet
+from itez.beneficiary.models import (
+Province,
+District,
+ServiceArea,
+WorkDetail
+)
+
+from itez.beneficiary.api.serializers import (
+    ProvinceSerializer,
+    DistrictSerializer,
+    ServiceAreaSerializer,
+    WorkDetailSerializer
+)
+
+
+class ProvinceList(
```

Comment:
calling it ProvincetList is limiting as this is handling CRUD properties
I would call it `ProvinceAPIView()`